src/Runtime/grafana-manifests/tests/Makefile

```
@@ -0,0 +1,46 @@
+.PHONY: setup-kind setup-grafana setup-grafana-dashboards clean
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)